### ISSUE TITLE
Cancel error in example

### ISSUE THREAD

**Issue by dynamicbutter** [ysak-y/flutter_audio_capture#3]
@ysak-y thanks for this lightweight audio capture project.  I found an issue in the example.  The problem occurs when trying to restart the example after it has been stopped.  On the second start I get the following error when running on iPhone 12 Pro version 14.2.1.  At this point I don't know if this is a problem in the example or the plugin.  I'm going to try to track down the root cause but was wondering if you may already know what it could be?

`flutter: PlatformException(ON_CANCEL_FAILED, Error occured in onCancel, null, null)`

**Comment by dynamicbutter** [comment]
The issue is in the plugin.  It is related to deactivating the audioSession before stopping.  Just swap the order of the calls to fix the problem.  Like so... 

```
diff --git a/ios/Classes/AudioCapture.swift b/ios/Classes/AudioCapture.swift
index 3f4bad19..0545ac31 100644
--- a/ios/Classes/AudioCapture.swift
+++ b/ios/Classes/AudioCapture.swift
@@ -61,7 +61,7 @@ class AudioCapture {
 
   public func stopSession() throws {
     let audioSession: AVAudioSession = AVAudioSession.sharedInstance()
-    try audioSession.setActive(false, options: .notifyOthersOnDeactivation)
     audioEngine.stop()
+    try audioSession.setActive(false, options: .notifyOthersOnDeactivation)
   }
 }
```
[pull request is here](https://github.com/ysak-y/flutter_audio_capture/pull/4)

**Comment by jasonletbetter** [comment]
https://github.com/ysak-y/flutter_audio_capture/pull/4

**Comment by ysak-y** [comment]
Merged #4, I'll publish new pub in a while.

**Comment by ysak-y** [comment]
Updated 😄 
https://pub.dev/packages/flutter_audio_capture/versions/0.0.3
